### PR TITLE
Remove bespoke table implementation entirely

### LINKED ISSUE #494: csvkit.utilities.csvsql still relies on bespoke Table implementation
Should use https://github.com/onyxfish/agate-sql

### PR DESCRIPTION
**Pull request by onyxfish**
Once complete this will close #494 and #496 and #497.

### CODE REVIEW

**Comment by onyxfish** [comment]
csvstat is basically done, though I need to go back through the logic to make sure I didn't screw up the output... also, the output format is kind of stupid, wouldn't hurt to clean that up.

File: tests/test_utilities/test_csvstat.py [hunk 1]
```diff
@@ -26,6 +31,7 @@ def test_encoding(self):
 
     def test_no_header_row(self):
         output = self.get_output(['-H', '-c', '2', 'examples/no_header_row.csv'])
+        print(output)
```
Comment (jpmckinney) [inline tests/test_utilities/test_csvstat.py]: Undo :)

File: CHANGELOG [hunk 2]
```diff
@@ -10,6 +10,9 @@ If you have built data workflows on top of csvkit you should not upgrade without
 
 Backwards-incompatible changes:
 
+* csvsql now generates DateTime columns instead of Time columns.
+* csvsql now generates Decimal columns instead of Integer, BigInteger, and Floatft columns.
```
Comment (jpmckinney) [inline CHANGELOG]: `Floatft` typo?

File: csvkit/cli.py [hunk 3]
```diff
@@ -73,7 +73,7 @@ def __init__(self, args=None, output_file=None):
         self.reader_kwargs = self._extract_csv_reader_kwargs()
         self.writer_kwargs = self._extract_csv_writer_kwargs()
 
-        self._install_exception_handler()
+        # self._install_exception_handler()
```
Comment (jpmckinney) [inline csvkit/cli.py]: Is this intentionally left commented out?

File: CHANGELOG (hunk 2 above)
Comment (onyxfish) [inline CHANGELOG]: Yep, fixed!

**Comment by onyxfish** [comment]
With upgrade to agate 1.5.2 fixing final Python 2 unicode issues, all tests are now passing.

**Pull request merged**